manifests/unit_file.pp

```diff
@@ -9,6 +9,25 @@
 ) {
   include ::systemd
 
+  if $title =~ /^([^\/]+)\// {
```

Comment:
I think this whole logic can be simplified by using a proper datatype for the parameter?

Reply:
AFAIK titles can't have data types.

Reply:
Ah right, not in the params section I think. But we can still use assert_type()?